rca/project_styleguide/templates/patterns/molecules/key-details/key-details.html

```diff
@@ -51,9 +51,9 @@ <h4 class="body body--two key-details__sub-heading">{{ programme_page_global_fie
                         {{ page.programme_details_time }} {{ page.get_programme_details_time_suffix_display }}
                     </li>
                     {% endif %}
-                    {% if page.programme_details_duration %}
+                    {% if page.study_mode %}
                     <li class="key-details__list-item">
-                        {{ page.get_programme_details_duration_display }}
+                        {{ page.study_mode }}
```

Comment:
Does this account for a page having 2 values tagged? How would 2 render here (double check with the client but I'm pretty sure they needed to support 2 tags of full time and part time)